### DISCUSSION TITLE
v1.1.4

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#265] [Announcements/公告]
**Full Changelog**: https://github.com/constanline/XQuickEnergy/compare/v1.1.3...v1.1.4

<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.1.4'>v1.1.4</a>.</em>

**Comment by qumingtaifeijin** [comment]
大佬开评论了！

**Comment by Stephentamm** [comment]
请问支付宝支持哪个版本，貌似步数和部分功能最新版本失效

**Reply by qumingtaifeijin** [reply]
我的是10.5.0其他人好像都比这个版本低。我这个版本到7点左右就不收能量，还提示操作异常。懒得换版本。除了这个其他都正常！

**Reply by Junfo** [reply]
不知道play版本支持不支持

**Comment by LiuYunGuiLang** [comment]
是不是支付宝版本的原因？我现在升级的最新版，目前是没有通知栏，而且有时候支付宝打开着但是不收能量，需要结束任务重新打开。这好像是一开始的毛病，在1.0正式版就修复了，然而现在又是这样了。中间更新1.12fix2的时候打开支付宝白屏卡死，我就更新支付宝到最新了，现在就是这样，保不住后台，没状态栏。

**Reply by LazyImmortal** [reply]
**最新版太广泛了，最新版是什么版？**
我的提问：
- 芝麻粒是否开启了状态栏禁删
- 支付宝是否给了常驻通知权限


我的测试：
- 关闭状态栏禁删，关闭常驻通知权限：有通知，但能被划掉
- 关闭状态栏禁删，开启常驻通知权限：有通知，但能被划掉
- 开启状态栏禁删，关闭常驻通知权限：无通知
- 开启状态栏禁删，开启常驻通知权限：有通知，不能被划掉

最后，@constanline 状态栏禁删应该改为通知栏禁删吧？

**Reply by LiuYunGuiLang** [reply]
![Screenshot_2023-08-24-13-18-55-445_com.miui.securitycenter.jpg](https://github.com/constanline/XQuickEnergy/assets/142814248/79bd53cb-815f-4b0d-a2aa-9af7fead8c55)

![Screenshot_2023-08-24-13-19-02-666_com.miui.securitycenter.jpg](https://github.com/constanline/XQuickEnergy/assets/142814248/7ae2cbc9-0a50-40cb-82b7-ffab3128c21f)

![Screenshot_2023-08-24-13-19-15-903_pansong291.xposed.quickenergy.repair.jpg](https://github.com/constanline/XQuickEnergy/assets/142814248/32646aab-3a14-4241-bfe0-bddd5e33df9e)

![Screenshot_2023-08-24-13-20-25-873_com.github.android.jpg](https://github.com/constanline/XQuickEnergy/assets/142814248/b7c85f67-a2b3-4ebf-b2ec-1e303824afb7)

最后一图为结束支付宝重新打开的状态栏。
芝麻粒版本1.1.5

**Reply by LiuYunGuiLang** [reply]
![Screenshot_2023-08-24-13-28-20-479_com.android.settings.jpg](https://github.com/constanline/XQuickEnergy/assets/142814248/c7cf23f4-cd77-4090-b81b-b07183c14093)

通知权限关闭重新打开，通知出现了，我再观察一下。

**Reply by LazyImmortal** [reply]
你图上不就显示你开启状态栏禁删了吗？我上面也说了，开启了状态栏禁删，需要给支付宝常驻通知权限才可以有通知。你要把常驻通知权限打开！！！
![Screenshot_2023-08-24-14-48-30-597_com.android.settings.jpg](https://github.com/constanline/XQuickEnergy/assets/44522979/a3de27ee-d269-4488-a64d-002da3254b67)

**Reply by LiuYunGuiLang** [reply]
常驻通知权限一直开着的，之前的版本没事。不过刚刚关闭重新打开就有了。俩小时了也还在，应该是好了。

**Comment by shaoying189** [comment]
这个最新版 应该是1.1.5版本

**Comment by WskDnpl** [comment]
这个版本的通知栏折叠了，上个版本不会的![Screenshot_20230824_131603.jpg](https://github.com/constanline/XQuickEnergy/assets/141890269/d97b2a9f-ca6d-46a5-876b-ef937f718927)

**Reply by hghghhh** [reply]
确实，我更新之后通知栏也被折叠了

**Reply by LazyImmortal** [reply]
这不正常吗？上个版本只有一条通知，这个版本有两条通知，自然会折叠起来。通知抽屉折叠起来还影响你使用吗？

**Reply by LazyImmortal** [reply]
这又不是bug

**Reply by hghghhh** [reply]
兄弟，我是跟水晶一起用的，芝麻粒通知展开不了

https://github.com/constanline/XQuickEnergy/assets/48682232/26f30135-800e-45ee-81fb-9791f250b091

**Reply by LazyImmortal** [reply]
> 兄弟，我是跟水晶一起用的，芝麻粒通知展开不了
> 
> https://github.com/constanline/XQuickEnergy/assets/48682232/26f30135-800e-45ee-81fb-9791f250b091
> 
> 

长按芝麻粒展开

**Reply by hghghhh** [reply]
确实有了，感谢🤗🤗

**Reply by WskDnpl** [reply]
> 这不正常吗？上个版本只有一条通知，这个版本有两条通知，自然会折叠起来。通知抽屉折叠起来还影响你使用吗？

有点影响的，上个版本跟这个版本显示的内容没变，折叠了需要点一下才能看见内容，没有折叠下拉就能看见了

**Comment by LiuYunGuiLang** [comment]
![IMG_20230824_152432.jpg](https://github.com/constanline/XQuickEnergy/assets/142814248/40eb6791-83a4-41b3-9ee6-48242b895e0a)

通知栏还在，有能量可收，但是不收能量。关闭支付宝再重新打开就收了。
支付宝版本10.5.10.8000
芝麻粒1.1.5![Screenshot_2023-08-24-15-26-31-610_com.eg.android.AlipayGphone-edit.jpg](https://github.com/constanline/XQuickEnergy/assets/142814248/c32024fd-09ea-459f-bb60-be5e640f5aa1)

**Reply by cwj851** [reply]
你后台掉了
